src/gluonnlp/data/transforms.py

```diff
@@ -1221,17 +1221,29 @@ class BERTSentenceTransform:
         Tokenizer for the sentences.
     max_seq_length : int.
         Maximum sequence length of the sentences.
+    vocab : Vocab or BERTVocab
```

Comment:
BERTVocab is a subclass of Vocab. Let's only mention Vocab here. It would be good to document what kind of Vocab special tokens / attributes are expected depending on the type of Bert model (BERT vs RoBERTa).